cdap-ui/app/features/hydrator/services/hydrator-service.js

```diff
@@ -80,7 +80,7 @@ class HydratorService {
     var params = {
       namespace: this.$state.params.namespace,
       pipelineType: appType,
-      version: (node.artifact && node.artifact.version ) || this.$rootScope.cdapVersion,
+      version: this.$rootScope.cdapVersion,
```

Comment:
@elfenheart This is used to for fetching properties of a plugin. We should always use cdap-etl-batch or cdap-etl-realtime for fetching properties. For fetching widgetJSON we need to have specific artifact names.